packages/blitz-rpc/src/client/invoke.ts

```diff
@@ -1,16 +1,36 @@
 import {FirstParam, PromiseReturnType, isClient, Ctx} from "blitz"
 import {RpcClient} from "./rpc"
 
-export function invoke<T extends (...args: any) => any, TInput = FirstParam<T>>(
+export async function invoke<T extends (...args: any) => any, TInput = FirstParam<T>>(
   queryFn: T,
   params: TInput,
-): Promise<PromiseReturnType<T>> {
+): Promise<T>
+export async function invoke<T extends (...args: any) => any, TInput = FirstParam<T>>(
+  queryFn: T,
+  params: TInput,
+  isServer: boolean,
+): Promise<T>
+export async function invoke<T extends (...args: any) => any, TInput = FirstParam<T>>(
+  queryFn: T,
+  params: TInput,
+  isServer = false,
```

Comment:
can we just use `typeof window` to determine `isServer` instead of requiring user to pass it in?

Reply:
@flybayer done